pagerditty/scripts.py

```diff
@@ -75,6 +75,9 @@ def generate_activity_intervals(since, until, engineer):
         report.ActivityInterval('incident', incident_pay),
     ]
 
+def _weekday(date):
```

Comment:
I believe you can get this from a date format string. I personally have my copy of the spreadsheet set up to reference the date column and format it to show the weekday, but I'm not against this change on that basis.

Reply:
that's fair enough, no need for this then